presto-docs/src/main/sphinx/sql/show-stats.rst

```diff
@@ -0,0 +1,31 @@
+==========
+SHOW STATS
+==========
+
+Synopsis
+--------
+
+.. code-block:: none
+
+    SHOW STATS FOR table
+    SHOW STATS FOR ( SELECT * FROM table [ WHERE condition ] )
+
+Description
+-----------
+
+Returns stats for the named table or for the results of a (limited) query.
+
+Stats are returned for each column, along with a rollup across all columns
+(the column will be ``NULL`` for this rollup).
```

Comment:
"rollup" ? The extra row has little to do with per-column rows.

Can we call this extra row a "summary row" or "table summary row" or "table stats"?

@electrum